public/index.html

```diff
@@ -11,13 +11,13 @@
     />
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
     <link
-      href="https://fonts.googleapis.com/css?family=Montserrat:400,500,700&display=swap"
+      href="https://fonts.googleapis.com/css?family=Montserrat:400,500,700|Bitter:400,700"
       rel="stylesheet"
     />
-    <link
+    <!-- <link
```

Comment:
Sorry, one last thing :-) It would be better to remove this line instead of commenting it out

Reply:
done.